<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>start</class>
 <widget class="QWidget" name="start">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushbutton_single">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>211</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_double">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>290</y>
     <width>211</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_help">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>290</y>
     <width>211</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>761</width>
     <height>531</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_background</zorder>
  <zorder>pushButton_help</zorder>
  <zorder>pushButton_double</zorder>
  <zorder>pushbutton_single</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
